lighthouse-extension/package-lock.json

```diff
@@ -0,0 +1,5979 @@
+{
```

Comment:
we're still yarn folk so we should be updating the yarn lock and not adding this, right?
or did you already point out why we need this and i'm forgetting?

Reply:
no no, it's my mistake . It shouldn't be here. I use yarn in WSL but npm on powershell because i'm to lazy to install yarn on windows as I only need it to run puppeteer (i'll probably create a bug for that in the puppeteer repository).

Reply:
fwiw `npm install -g yarn` is what we do on appveyor. probably not too bad.

Reply:
also i think pptr is moving to package-lock soon.